### PR TITLE
Usability improvements trending_tags.tpl & CSS instructions for formatting in the notification.tpl 

### PR DESCRIPTION
**Pull request by loma-one**
The tranding tags don't look very nice. The changes are intended to make them more attractive. 

 - For this purpose, each tag is preceded by a "fa fa-hashtag".
 - The # in front of the word has been removed
 - Clicking on a hashtag searches with a hashtag as usual.
 - Other tags were previously displayed indented
 - Now it is ensured that all hashtags are in one line

This is a suggestion that can be discarded at any time. I would still be happy if the changes could be adopted.

 **CSS instructions for formatting in the notification.tpl**

A space has been inserted between the user icon and the outlined text. Texts are wrapped after 70 characters.

### CODE REVIEW

File: view/templates/notifications/notification.tpl [hunk 1]
```diff
@@ -1,4 +1,29 @@
 
+<style>
+	/* Flexbox layout to align the icon and text in a single line */
```
Comment (annando) [inline view/templates/notifications/notification.tpl]: Please add styles in the general css file.

Comment (loma-one) [reply view/templates/notifications/notification.tpl]: The separation has taken place. I hope it's true.

File: view/templates/widget/trending_tags.tpl [hunk 2]
```diff
@@ -5,22 +5,59 @@
...
+	var moreTags = document.getElementById('more-tags');
+	var link = event.target.closest('a');
+	var caretIcon = document.getElementById('caret-icon');
+	var linkText = document.getElementById('link-text');
+
+	if (moreTags.style.display === 'none') {
+		moreTags.style.display = 'block';
+		linkText.textContent = 'Show Less';
+		link.setAttribute('aria-expanded', 'true');
+		caretIcon.className = 'fa fa-caret-down'; // Ändert das Icon auf "Caret Down"
```
Comment (annando) [inline view/templates/widget/trending_tags.tpl]: Please use English comments.

File: view/templates/notifications/notification.tpl (hunk 1 above)
Comment (loma-one) [reply view/templates/notifications/notification.tpl]: Oh yes. Thank you. I had overlooked that.

File: view/templates/widget/trending_tags.tpl (hunk 2 above)
Comment (loma-one) [reply view/templates/widget/trending_tags.tpl]: Oh yes. Thank you. I had overlooked that.

File: view/templates/widget/trending_tags.tpl [hunk 3]
```diff
@@ -5,22 +5,59 @@
...
+	if (moreTags.style.display === 'none') {
+		moreTags.style.display = 'block';
+		linkText.textContent = 'Show Less';
+		link.setAttribute('aria-expanded', 'true');
+		caretIcon.className = 'fa fa-caret-down'; // Changes the icon to "Caret Down"
+	} else {
+		moreTags.style.display = 'none';
+		linkText.textContent = 'Show More';
+		link.setAttribute('aria-expanded', 'false');
+		caretIcon.className = 'fa fa-caret-right'; // Changes the icon to "Caret Down"
```
Comment (annando) [inline view/templates/widget/trending_tags.tpl]: Nitpicker here :grimacing: I guess that icon isn't "caret down"?

Comment (loma-one) [reply view/templates/widget/trending_tags.tpl]: I have removed the comments. They were only meant for me. They are no longer relevant.

**Pull request merged**